typing/typemod.ml

```diff
@@ -164,7 +164,7 @@ let initial_env ~loc ~safe_string ~initially_opened_module
       env
   in
   let units =
-    List.rev_map Env.persistent_structures_of_dir (Load_path.get ())
+    List.map Env.persistent_structures_of_dir (Load_path.get ())
```

Comment:
This is the only place I could find where the caller was correctly reversing the result of `Load_path.get` before using it. After the changes in the PR, the results are being returned in normal order, so this reversal is no longer needed.